rclcpp/wait_set/README.md

```diff
@@ -0,0 +1,22 @@
+# Minimal rclcpp wait-set cookbook recipes
+
+This package contains a few different strategies for creating nodes which use `rclcpp::waitset`
+to wait and handle ROS entities, that is, subscribers, timers, clients, services, guard
+conditions and waitables.
```

Comment:
Let's add a note about when and why you might want to create applications like this, i.e. you probably only want to do this in very specific situations, or not at all (in which case these are more demonstrations than examples you should copy-paste-edit).

Reply:
Added a note

Reply:
Where did you add it? I still just see this paragraph and the list of nodes below.

Reply:
I added it in the `minimal_subscriber` README. Do I move the note here?

>The following examples `wait_set_subscriber.cpp`, `static_wait_set_subscriber.cpp` and 
`time_triggered_wait_set_subscriber.cpp` show how to use a subscription in a node using a `rclcpp` 
wait-set. This is not a common use case  in ROS 2 so this is not the recommended strategy to 
use by-default. This strategy makes sense in some specific situations, for example when the 
developer needs to have more control over callback order execution, to create custom triggering 
conditions or to use the timeouts provided by the  wait-sets.   

https://github.com/ros2/examples/pull/315/files#diff-7cfbf34595c2e8d6c230d5ee9d207ed0deca9f060b26da8334b512b9755f7b65R20-R25